sceptre/template.py

```diff
@@ -261,3 +263,30 @@ def _create_bucket(self, region, bucket_name, connection_manager):
                     }
                 }
             )
+
+
+def _render_jinja_template(template_dir, filename, jinja_vars):
```

Comment:
Compromise with static method - another PR for module functions and global loggers.